static/app/views/replays/detail/network/networkHeaderCell.tsx

```diff
@@ -24,7 +24,26 @@ const COLUMNS: {
   label: ReactNode;
 }[] = [
   {field: 'method', label: t('Method')},
-  {field: 'status', label: t('Status')},
+  {
+    field: 'status',
+    label: (
+      <Fragment>
+        {t('Status')}
+        <Tooltip
+          title={tct(
+            'If the status is [zero], the resource might be a cross-origin request.[linebreak][linebreak] Configure your CDN to respond with the CORS header [header] to so supported browsers can report the real status code.',
```

Comment:
```suggestion
            'If the status is [zero], the resource might be a cross-origin request.[linebreak][linebreak] Configure your CDN to respond with the CORS header [header] so supported browsers can report the real status code.',
```

Reply:
maybe s/real/actual/?

```suggestion
            'If the status is [zero], the resource might be a cross-origin request.[linebreak][linebreak] Configure your CDN to respond with the CORS header [header] so supported browsers can report the actual status code.',
```